README.md

```diff
@@ -1,12 +1,21 @@
 # DSC Devsprints Jan 2020
 ## Date and Venue - TBD
+#Project Suggestion - Here are some of the links of the projects I found interesting:
+
+https://summerofcode.withgoogle.com/archive/2019/projects/5798435662331904/
+
+https://summerofcode.withgoogle.com/archive/2019/projects/6316883983204352/
+
+https://summerofcode.withgoogle.com/archive/2019/projects/4513054174216192/
+
+https://summerofcode.withgoogle.com/archive/2019/projects/6203072802455552/
```

Comment:
You need to edit the markdown table to add the project name. Please do not add GSoC problem statements as projects. These projects have already been completed.